Makefile

```diff
@@ -5,7 +5,7 @@ COSMOS_BUILD_OPTIONS ?= ""
 PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
 PACKAGES_SIM=github.com/quicksilver-zone/quicksilver/test/simulation
 PACKAGES_E2E=$(shell go list ./... | grep '/e2e')
-VERSION=$(shell git describe --tags | head -n1)
+VERSION=v1.5.5
```

Comment:
why is version being hardcoded here?

Reply:
`git describe --tags | head -n1` return `fe/v1.1.8-1-g89b98daf`

Reply:
because that was the last version to be tagged; but we can't hardcode in the Makefile.

We should probably change this to `git describe --tags | grep -v 'fe/' | head -n1`